packages/umpire/package.py

```diff
@@ -275,6 +276,7 @@ def initconfig_package_entries(self):
         entries.append(cmake_cache_path("BLT_SOURCE_DIR", spec["blt"].prefix))
         if spec.satisfies("@5.0.0:"):
             entries.append(cmake_cache_path("camp_DIR", spec["camp"].prefix))
+        entries.append(cmake_cache_path("fmt_DIR", spec["fmt"].prefix))
```

Comment:
Should only append fmt_DIR `if spec.satisfies("@develop")`?

Reply:
Then when Umpire is next released, we can update the check to a specific version number.

Reply:
I _think_ `spec["fmt"].prefix` will only return a path if fmt is in the spec (if umpire is @ develop) and an empty path otherwise.

So, I would say it's not mandatory to put the conditional here too.